tools/wpt/expectation.json

```diff
@@ -3153,20 +3153,6 @@
         "queue-microtask.any.html": true,
         "queue-microtask.any.worker.html": true
       },
-      "scripting": {
-        "reporterror.any.html": [
-          "self.reportError(1)",
-          "self.reportError(TypeError)",
-          "self.reportError(undefined)",
-          "self.reportError() doesn't invoke getters"
-        ],
-        "reporterror.any.worker.html": [
-          "self.reportError(1)",
-          "self.reportError(TypeError)",
-          "self.reportError(undefined)",
-          "self.reportError() doesn't invoke getters"
-        ]
-      },
```

Comment:
This doesnt seem right; we should still test these, not completely remove the tests